temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -273,6 +292,83 @@ public Builder setSslContext(SslContext sslContext) {
       return this;
     }
 
+    /**
+     * Sets default SSL context parameters that can be used with the temporal cloud service. Users
+     * that require additional customization may use {@link #setSslContext(SslContext)} directly.
+     *
+     * @param keyCertChainInputStream - an input stream for an X.509 certificate chain in PEM
```

Comment:
we should state that this is the "client" certificate chain to avoid confusion. similar for keyInputStream, we should mention it is the client certificate's private key

Reply:
Parameters and comments map 1:1 with what we have in netty, see [netty documentation](https://netty.io/4.0/api/io/netty/handler/ssl/SslContextBuilder.html) for details.